build.gradle

```diff
@@ -94,14 +94,21 @@ ext.libraries = [
     mockito: 'org.mockito:mockito-core:3.3.3',
     netty: 'io.netty:netty-all:4.1.52.Final',
     oss: 'org.sonatype.oss:oss-parent:7',
+    pulsarClient: "${pulsarGroup}:pulsar-client:${pulsarVersion}",
+    pulsarIoCore: "${pulsarGroup}:pulsar-io-core:${pulsarVersion}",
+    pulsarIoCommon: "${pulsarGroup}:pulsar-io-common:${pulsarVersion}",
     r2: 'com.linkedin.pegasus:r2:' + pegasusVersion,
     restliCommon: 'com.linkedin.pegasus:restli-common:' + pegasusVersion,
     rocksdbjni: 'org.rocksdb:rocksdbjni:7.0.3',
     samzaApi: 'org.apache.samza:samza-api:1.5.1',
+    slf4j: 'org.slf4j:slf4j:1.7.36',
+    slf4jApi: 'org.slf4j:slf4j-api:1.7.36',
+    slf4jSimple: 'org.slf4j:slf4j-simple:1.7.36',
     snappy: 'org.iq80.snappy:snappy:0.4',
     spark: 'com.sparkjava:spark-core:2.9.4',
     spotbugs: 'com.github.spotbugs:spotbugs:4.5.2',
     tehuti: 'io.tehuti:tehuti:0.9.2',
+    testcontainers: 'org.testcontainers:testcontainers:1.18.0',
```

Comment:
Thanks for providing a demo on how to leverage test containers. I've been meaning to try it for a while. Looks useful.